Source/IntegrationTests/TestFiles/LitTests/LitTest/comp/rust/externalclasses.dfy

```diff
@@ -0,0 +1,103 @@
+// NONUNIFORM: Temporary development of the Rust compiler
+// RUN: %baredafny run --target=rs --input "%S/externalclasses.rs" "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+module {:extern "External.Class.Container"} ExternalClassContainer {
+  class {:extern} ExternalClass {
+    constructor {:extern} (i: int)
+  }
+  method {:extern} Test() returns (i: int)
+  trait GetValueHolder {
+    function GetValue(): int
+  }
+
+  class {:extern} ExternalPartialClass extends GetValueHolder {
+    constructor {:extern} ()
+    function {:extern} GetValue(): int
+    method Print() {
+      print GetValue();
+    }
+  }
+  class {:extern} ExternalPartialClass2 extends GetValueHolder {
+    constructor {:extern} ()
+    function GetValue(): int {
+      2
+    }
+    method Print() {
+      print GetValue();
+    }
+  }
+}
+
+module Dafny.FileIO {
```

Comment:
If this implementation is working, why not move it to the actual DafnyStandardLibraries?? (Can be in a follow-up PR for sure, but worth prioritizing)

Reply:
This is only a mock implementation of Dafny.FileIO. The extern actually don't read the file system. But it gives an idea on how to achieve that. Let's think of a proper standard library in an upcoming PR.